cmd/object-api-errors.go

```diff
@@ -666,6 +673,12 @@ func isErrVersionNotFound(err error) bool {
 	return errors.As(err, &versionNotFound)
 }
 
+// isErrVersionNotFound - Check if error type is SignatureDoesNotMatch.
+func isErrSignatureDoesNotMatch(err error) bool {
+	var signatureDoesNotMatch SignatureDoesNotMatch
+	return errors.As(err, &signatureDoesNotMatch)
+}
```

Comment:
seems like left over from rebase conflicts - master no longer has this function

Reply:
This is function is used to identify a signature mismatch error and translate it to "invalid tier credentials".